portal-api/src/v1/email/index.test.js

```diff
@@ -23,65 +23,62 @@ describe('sendEmail', () => {
+    mockExternalApiEmail.mockImplementation(() => Promise.resolve(externalApiSendEmailResponse));
 
     const response = await sendEmail(templateId, emailAddress, emailVariables);
 
-    expect(response).toEqual(notifyClientSendEmailResponse);
+    expect(response).toEqual(externalApiSendEmailResponse);
   });
 
-  it('returns an object with a 500 status code if the sendEmail method call on the notifyClient fails and the error object has a different status code', async () => {
+  it('returns an object with a 500 status code if the sendEmail method call on the externalApi fails and the error object has a different status code', async () => {
     const error = { response: { status: 400 } };
     const expectedResponse = { status: 500, data: 'Failed to send an email' };
 
-    mockNotifyClientSendEmail.mockImplementation(() => Promise.reject(error));
+    mockExternalApiEmail.mockImplementation(() => Promise.reject(error));
 
     const response = await sendEmail(templateId, emailAddress, emailVariables);
 
     expect(response).toEqual(expectedResponse);
   });
 
-  it('returns an object with a 500 status code if the sendEmail method call on the notifyClient fails and the error object has no status code', async () => {
+  it('returns an object with a 500 status code if the sendEmail method call on the externalApi fails and the error object has no status code', async () => {
     const error = {};
     const expectedResponse = { status: 500, data: 'Failed to send an email' };
 
-    mockNotifyClientSendEmail.mockImplementation(() => Promise.reject(error));
+    mockExternalApiEmail.mockImplementation(() => Promise.reject(error));
 
     const response = await sendEmail(templateId, emailAddress, emailVariables);
 
     expect(response).toEqual(expectedResponse);
   });
 
-  it('prints an error to the console including the data from the error object from the sendEmail method call on the notifyClient, if it fails', async () => {
+  it('prints an error to the console including the data from the error object from the sendEmail method call on the externalApi, if it fails', async () => {
     const someArray = ['some data', 'some more data'];
     const error = { response: { data: someArray } };
 
-    mockNotifyClientSendEmail.mockImplementation(() => Promise.reject(error));
+    mockExternalApiEmail.mockImplementation(() => Promise.reject(error));
 
     await sendEmail(templateId, emailAddress, emailVariables);
 
     expect(console.error).toHaveBeenCalledTimes(1);
     expect(console.error).toHaveBeenCalledWith('Portal API - Failed to send email %o', error?.response?.data);
   });
 
-  it('prints an error to the console if the sendEmail method call on the notifyClient fails and the error object has no data', async () => {
+  it('prints an error to the console if the sendEmail method call on the externalApi fails and the error object has no data', async () => {
```

Comment:
Can we please add `429` assertion. Thanks

Reply:
This helper unit test where we mock externalApi call.

It is no logic related to status codes, just 400 and 500 responses on errors.

So it is nothing to test for 429 response.